microservices/composite/Dockerfile

```
@@ -1,4 +1,15 @@
-FROM eclipse-temurin:17
-EXPOSE 8080
+FROM openjdk:17-jdk-slim as builder
+WORKDIR application
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)